src/index.html

```diff
@@ -11,12 +11,61 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MoYo"
+        />
+      </a>
```

Comment:
The `<a>` tag used for the logo link has an `href` attribute set to `#`, which is a placeholder. Consider replacing `#` with a meaningful URL or using `javascript:void(0);` if the link is not intended to navigate anywhere.